docs/walkthroughs/installing-slate.md

```diff
@@ -12,6 +12,8 @@ You'll also need to be sure to install Slate's peer dependencies:
 npm install react react-dom immutable
 ```
 
+_Note, if you are new to react, then read the [react introduction](https://reactjs.org/docs/try-react.html) first and don't forget to install def-dependencies like babel._
```

Comment:
*`dev-dependencies`

Reply:
also, if someone is new to React, we should recommend them to use create-react-app so that they don't have to install anything like Babel